kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/bluetooth/listener/BluetoothAdvertisementScanListener.java

```diff
@@ -0,0 +1,18 @@
+package org.eclipse.kura.bluetooth.listener;
```

Comment:
1. Missing License header
2. Missing package-info.java in ``org.eclipse.kura.bluetooth.listener``